packages/contact-center/task/src/task.types.ts

```diff
@@ -0,0 +1,89 @@
+import {ITask, IContactCenter} from '@webex/plugin-cc';
+
+/**
+ * Interface representing the TaskProps of a user.
+ */
+export interface TaskProps {
+  /**
+   * The name of the user.
+   */
+  currentTask: ITask;
+
+  /**
+   * Webex instance.
+   */
+  cc: IContactCenter;
+
+  /**
+   * Handler for task accepted
+   */
+  onAccepted?: () => void;
+
+  /**
+   * Handler for task declined
+   */
+  onDeclined?: () => void;
+
+  /**
+   * accept incoming task action
+   */
+  accept: () => void;
+
+  /**
+   * decline incoming task action
+   */
+  decline: () => void;
+
+  /**
+   * Flag to determine if the user is logged in with a browser option
+   */
+  isBrowser: boolean;
+
+  /**
+   * Flag to determine if the task is answered
+   */
+  answered: boolean;
```

Comment:
Since this is a boolean let's stick to the same format as the previous variable
```suggestion
  isAnswered: boolean;
```

Reply:
addressed